app/src/main/java/com/travelbetadisaster/travel_log/database/tables/UserHistory.kt

```diff
@@ -8,23 +8,44 @@ import java.util.Date
 class UserHistory {
     @PrimaryKey(autoGenerate = true)
     @ColumnInfo(name = "historyId")
-    val historyId: Int = 0
+    var historyId: Int = 0
 
     @ColumnInfo(name = "userId")
-    val userId: Int = 0
+    var userId: Int = 0
 
     @ColumnInfo(name = "activityType")
-    val activityType: String? = null
+    var activityType: String? = null
 
     @ColumnInfo(name = "activityDate")
-    val activityDate: Date? = null
+    var activityDate: Date? = null
 
     @ColumnInfo(name = "description")
-    val description: String? = null
+    var description: String? = null
 
     @ColumnInfo(name = "totalMiles")
-    val totalMiles: Double = 0.0
+    var totalMiles: Double = 0.0
 
     @ColumnInfo(name = "visitId")
-    val visitId: Int = 0
+    var visitId: Int = 0
+
+    constructor() {}
+
+    constructor(id: Int, userId: Int, activityType: String, activityDate: Date, description: String, totalMiles: Double, visitId: Int) {
```

Comment:
Completely forgot activity type is something we are storing. I'll look into adding an activity type dropdown into the journal entry fragments